### DISCUSSION TITLE
Feature request: improve Font Awesome Pro Subsetter usability

### DISCUSSION THREAD

**Discussion by MrUpsidown** [FortAwesome/Font-Awesome#20937] [Feature Requests]
### Is your feature request related to a problem?

The Font Awesome Pro Subsetter is barely usable. Please improve UX in your product.

### Feature description

* When searching for an icon and selecting it from the results, why show Solid THEN Regular THEN Light, etc. ? Why not simply show the solid version and let us select the various versions once the icon has been selected?
* When checking what icons are in an existing set (right sidebar) I have NO WAY to know the name of the icon. Clicking on it will remove the icon from the set, hovering the mouse cursor over the icon just displays "remove" text. Why not simply display the name of the icon along like "remove circle-check" for example? That would at least allow me to 1) remove the icon and more importantly 2) know the name of the icon I am looking at.
* When searching for icons, why not allow alternative icon names / keywords to lead to the icons? For example, searching for "house" leads to dozens of results but searching for "home" leads to NONE. Also, searching for any (existing) icon name but with a space after it leads to no results, for example, searching for "check" leads to many results while searching for "check " (with a space) leads to NONE. Searching for "checkmark" or "check mark" leads to NO RESULTS while searching for "check" leads to many icons with... a check mark.

### Alternatives

I am not aware of alternatives.

### Feature request checklist

- [X] The title starts with "Feature request: " and is followed by the requested feature description
- [X] This is a single feature
- [X] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate

**Comment by tagliala** [comment]
Hi @MrUpsidown, do you have access to the Pro repository at https://github.com/fortawesome/Font-Awesome-Pro?

If you don't have access, please make sure you have linked your Font Awesome Account to your GitHub account at the very bottom of https://fontawesome.com/account

Please just confirm that you have access, I would like to move there this issue

**Comment by MrUpsidown** [comment]
That github link you provided leads to a 404. I can't seem to see any way to link my FontAwesome account with my Github account :/

**Comment by tagliala** [comment]
Ok apologies, I now see that it is no longer necessary.

So I will not move this request to the other repo and I've already asked who better knows the Pro Subsetter to share their thoughts here

**Comment by robmadole** [comment]
Thanks @MrUpsidown . Appreciate you taking the time to send this in. We'll put this on the list to think through.

**Comment by mbaas2** [comment]
Just wanted to submit a FR myself for the hovering. As it is still not implemented, I'd like to add my humble voice in its favour😉
